layouts/partials/footer.html

```diff
@@ -0,0 +1,137 @@
+                <li class="leaf"><a href="https://www.eclipse.org/legal/termsofuse.php" title="">Terms of Use</a></li>
+                <li class="leaf"><a href="https://www.eclipse.org/legal/copyright.php" title="">Copyright Agent</a></li>
+                <li class="leaf"><a href="https://eclipse.org/org/documents/epl-v10.php" title="">Eclipse Public License</a></li>
+                <li class="last leaf"><a href="https://www.eclipse.org/legal/" title="">Legal Resources</a></li>
+              </ul>
+            </div>
+          </section> <!-- /.block -->
+        </div>
+      </section>
+
+      <section id="footer-useful-links" class="col-xs-offset-1 col-xs-11 col-sm-7 col-md-6 col-md-offset-0 hidden-print">
+        <div class="region region-footer-3 solstice-region-element-count-1 row">
+          <section id="block-menu-menu-locationtech-useful-links" class="block block-menu block-region-footer-3 block-menu-locationtech-useful-links clearfix">
+          <h2 class="block-title">Useful Links</h2>
+            <div class="block-content">
+              <ul class="menu nav">
+                <li class="first leaf"><a href="https://bugs.eclipse.org/bugs/" title="">Report a Bug</a></li>
+                <li class="leaf"><a href="https://help.eclipse.org/" title="">Documentation</a></li>
+                <li class="leaf"><a href="https://eclipse.org/contribute/" title="">How to Contribute</a></li>
+                <li class="leaf"><a href="https://eclipse.org/mail/" title="">Mailing Lists</a></li>
+                <li class="leaf"><a href="https://eclipse.org/forums/" title="">Forums</a></li>
+                <li class="last leaf"><a href="https://marketplace.eclipse.org/" title="">Marketplace</a></li>
+              </ul>  
+            </div>
+          </section> <!-- /.block -->
+        </div>
+      </section>
+
+      <section id="footer-other" class="col-xs-offset-1 col-xs-11 col-sm-7 col-md-6 col-md-offset-0 hidden-print">
+        <img class="logo-eclipse-default img-responsive" width="140" typeof="foaf:Image" src="https://www.openmdm.org/sites/all/themes/solstice/_themes/solstice_openmdm/logo.png" alt="" title=""><p class="padding-top-15">OpenMDM is a Working Group of The Eclipse Foundation.</p><hr><a href="https://www.eclipse.org"><img class="logo-eclipse-default img-responsive" width="160" typeof="foaf:Image" src="https://www.openmdm.org/sites/all/themes/solstice/public/images/logo/eclipse-foundation-grey-orange.svg" alt="" title="Eclipse Foundation"></a>
+      </section>
+      <div class="col-sm-24 margin-top-20">
+        <div class="row">
+          <div id="copyright" class="col-md-16">
+            <p id="copyright-text">Copyright © Eclipse Foundation, Inc. All Rights Reserved.</p>
+          </div>
+          <div class="col-md-8 social-media">
+            <ul class="list-inline">
+              <li>
+                <a class="social-media-link fa-stack fa-lg" href="https://twitter.com/EclipseFdn">
```

Comment:
Check the socials.html partial in base theme, we should probably use that here (if possible)